styles/style-main.css

```diff
@@ -0,0 +1,103 @@
+body {
+  background-image: url(../images/mobile_geometry_1.png);
+  background-repeat: no-repeat;
+}
+
+.header {
+  display: flex;
+  border: solid red;
+}
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+  margin-bottom: 50px;
+}
+
+.toggleNav {
+  display: block;
+  justify-self: flex-end;
+}
+
+.navbar ul {
+  display: none;
+
+  /* SAVED CONFIG FOR DROP DOWN MENU ITEMS
+  display: flex;
+  flex-direction: column;
+  align-self: center;
+  align-items: center;
+  padding-inline-start: 0px;
+  */
+}
```

Comment:
- [ ] I can see you want to save this piece of code for future use 👍. However, let's imagine a recruiter inspecting your work right now and seeing these commented lines of code would not make your work look professional. I think it would be better to move them to your notepad or sticky notes and save them there then you can get them when you need to make use of them in your work 😃